pontoon/base/models.py

```diff
@@ -3326,3 +3334,73 @@ def calculate_stats(self, save=True):
             strings_with_warnings_diff,
             unreviewed_strings_diff,
         )
+
+
+from pontoon.machinery.views import translation_memory
+from pontoon.machinery.views import google_translate
+
+
+def pre_translate(project, locales=None, entities=None):
```

Comment:
We should move this outside `pontoon.base.models`. I suspect we'll need a dedicated module, e.g. `pontoon.pre_translation`

Reply:
Agreed. I don't much like the underscore in that name though, I'd simply call it `pretranslation`.